runtime/runtime_test.go

```diff
@@ -7548,24 +7550,36 @@ func TestRuntimeComputationMetring(t *testing.T) {
               i = i + 1
           }
         `,
-			ok:          false,
-			expCompUsed: compLimit,
+			ok:        false,
+			hits:      compLimit,
+			intensity: 6,
 		},
 		{
 			name: "statement + createArray + transferArray + too many for-in loop iterations",
 			code: `
           for i in [1, 2, 3, 4, 5, 6, 7, 8, 9, 10] {}
         `,
-			ok:          false,
-			expCompUsed: compLimit,
+			ok:        false,
+			hits:      compLimit,
+			intensity: 15,
 		},
 		{
-			name: "statement + createArray + transferArray + some for-in loop iterations",
+			name: "statement + createArray + transferArray + two for-in loop iterations",
 			code: `
           for i in [1, 2] {}
         `,
-			ok:          true,
-			expCompUsed: 5,
+			ok:        true,
+			hits:      5,
+			intensity: 6,
+		},
+		{
+			name: "statement + (functionInvocation x 2) + encoding",
+			code: `
+          acc.save("A quick brown fox jumps over the lazy dog", to:StoragePath(identifier: "some_path")!)
```

Comment:
Maybe just use a path literal

```suggestion
          acc.save("A quick brown fox jumps over the lazy dog", to: /storage/some_path)
```